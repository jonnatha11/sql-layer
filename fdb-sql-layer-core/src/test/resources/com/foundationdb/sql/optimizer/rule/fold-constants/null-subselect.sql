SELECT name, (SELECT order_date FROM orders
               WHERE 1=0) AS v
 FROM customers
