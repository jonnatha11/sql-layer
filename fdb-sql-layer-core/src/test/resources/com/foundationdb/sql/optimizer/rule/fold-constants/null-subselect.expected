SelectQuery@691177
  ResultSet@6defe475[name, v]
    Project@207ff5b6[customers.name, NULL]
      Select@13043d2[]
        TableSource@d55986(customers)
